feature-libs/commerce-quotes/occ/adapters/occ-commerce-quotes.adapter.spec.ts

```diff
@@ -0,0 +1,312 @@
+// import { HttpClient } from "@angular/common/http";
```

Comment:
`should:` remove comment

Reply:
already removed with [ef44817](https://github.com/SAP/spartacus/pull/15742/commits/ef44817571204b0b95783311a669079033338612) commit